@here/harp-mapview/lib/Tile.ts

```diff
@@ -434,6 +437,14 @@ export class Tile implements CachedResource {
 
     private m_colorMap: Map<string, THREE.Color> = new Map();
 
+    private m_extrudedObjects: Array<THREE.Mesh | THREE.LineSegments> = [];
```

Comment:
All this stuff should be moved to already mentioned `AnimatedExtrusionTileHandler` as commented above.

With related methods of course.